src/Web/aspnet-core/src/Piggyvault.Web.Host/appsettings.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "Default": "Server=localhost; Database=PiggyvaultDb; Trusted_Connection=True;"
+    "Default": "Server=localhost; Database=PiggyvaultDb; Trusted_Connection=False; User ID=sa; Password=P@55w0rd"
```

Comment:
I think this can be excluded from PR. (Keep this change in your local)

Reply:
I have development version, but I changed the file root because I needed to run EFCore Mogrations.